R/rmarkdown.R

```diff
@@ -50,3 +53,15 @@ render_rmarkdown <- function(pkg, input, output, ..., copy_images = TRUE, quiet
 
   invisible(path)
 }
+
+#' Get BSTINPUTS for JSS articles (adapted from tools::texi2dvi)
+#'
+#' @noRd
+get_bstinputs <- function(input_path) {
+  Rbstinputs <- file.path(R.home("share"), "texmf", "bibtex", "bst")
+  obstinputs <- Sys.getenv("BSTINPUTS", unset = NA_character_)
+  if(is.na(obstinputs))
+    obstinputs <- "."
+
+  paste(obstinputs, dirname(input_path), Rbstinputs, sep = .Platform$path.sep)
```

Comment:
Please use `path()` and `path_dir()`

Reply:
Also I don't understand the logic here - why are you combining an absolute path (`Rbstinputs`) with a potentially relative path?

Reply:
This combines the old value of BSTINPUTS (empty or some path provided by the user), the input path (just in case the user placed some BST file along with the article) and the path where the `jss.bst` lives. The result is something like `some/path:another/path:/usr/blablabla/R/etcetera`. The `.Platform$path.sep` is `:` for Linux.

Reply:
Well, then the `"."` isn't really needed here.

Reply:
Oooh, I see now. I think it would be clear to make a vector of paths and then use `collapse`

Reply:
Ok, changed.